examples/Examples/Database/Mongo/Find.purs

```diff
@@ -10,24 +10,14 @@ import Control.Monad.Eff.Exception
 
 import Data.Argonaut (printJson)
 import Data.Argonaut.Core (Json(..))
-import Data.Argonaut.Decode (DecodeJson, decodeJson)
+-- import Data.Argonaut.Decode (DecodeJson, decodeJson)
```

Comment:
Since these examples are broken and I've basically converted all the examples into test cases, do you think we still need them?

We could just add a link to Readme.MD, saying "please check out the tests for more examples?"